UnitsNet/UnitConverter.cs

```diff
@@ -54,7 +40,7 @@ public static double Convert(QuantityValue fromValue, Enum fromUnitValue, Enum t
         public static bool TryConvert(QuantityValue fromValue, Enum fromUnitValue, Enum toUnitValue, out double convertedValue)
         {
             convertedValue = 0;
-            if (!Quantity.TryFrom(fromValue, fromUnitValue, out IQuantity from)) return false;
+            if (!Quantity.TryFrom(fromValue, fromUnitValue, out var from)) return false;
```

Comment:
I'd add back `IQuantity` here, it's not obvious to me what type `from` is.

Reply:
See above. If you agree we can leave, if not I'll add it back!

Reply:
I did rename this ```fromQuantity``` to match the other rename you suggested. That should make it more obvious too.

Reply:
I would still prefer to add it back here.

Reply:
Done!